### PR TITLE
New Issue Template

### PR DESCRIPTION
**Pull request by ashmaroli**
Drafted a new template for submitting Issues.
- Commented out any existing hints.
- Added placeholder texts for certain sections.
- Added descriptive sub-headings.

### CODE REVIEW

**Comment by ashmaroli** [comment]
I've opened a new issue ( No. 197 ) with this template to additionally serve as a live preview for this PR.

**Comment by ashmaroli** [comment]
@mertkahyaoglu, I think `documentation` is actually an incorrect label here. This PR doesn't affect JekyllAdmin documentation, only the Issue drafting here on GitHub.  
A better alternative would be `GitHub Repo` or similar.  :smiley:

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
-Describe in one or two sentences why you're opening this issue:
+### Description:
+<!-- Describe in one or two sentences why you're opening this issue: -->
+> **REPLACE-ME**
```
Comment (benbalter) [inline .github/ISSUE_TEMPLATE.md]: Why should the issue description be a block quote? What are they quoting?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 
+### Tell us a bit about yourself:
+* Version of JekyllAdmin I'm using <HINT: use `bundle show` to check>: ***REPLACE-ME***
```
Comment (benbalter) [inline .github/ISSUE_TEMPLATE.md]: Why add the `**REPLACE ME**` text? I feel like that's creating more work for a bug reporter (highlight and delete 5 times), than just having a colon and a blank which should achieve the same effect.

**Comment by benbalter** [comment]
Thanks @ashmaroli! It may also be worth noting, either in this PR or a subsequent follow up, that you should report *one* bug per issue, not multiple, where they can be distinguished.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (ashmaroli) [inline .github/ISSUE_TEMPLATE.md]: :smile: themselves? lol!
I placed it there for the formatting. But I now realize why that's wrong. So, I'll remove the blockquote formatting.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (ashmaroli) [inline .github/ISSUE_TEMPLATE.md]: I wanted the issue drafter to insert the info exactly between the three stars to have their response formatted. Rarely does a user style their issue draft.
I'll remove those too..

**Comment by ashmaroli** [comment]
uhm.. this is one, single issue.
I understand what you mean though.

**Comment by benbalter** [comment]
> It may also be worth noting, either in this PR or a subsequent follow up, that you should report one bug per issue, not multiple, where they can be distinguished.

Don't have to do it here, I was suggesting that we add a note with that reminder to the form (not that you were breaking the rule!) :smile:

**Pull request merged**